validator/src/bin/solana-test-validator.rs

```diff
@@ -21,6 +21,7 @@ use {
         signature::{read_keypair_file, write_keypair_file, Keypair, Signer},
         system_program,
     },
+    solana_streamer::socket::SocketAddrSpace,
```

Comment:
I think `solana-test-validator` should just allow private addresses by default (and it's probably fine to not have an option to only allow global this).

Reply:
yeah, true. I am merging this now not to get merge conflicts, but will follow up.